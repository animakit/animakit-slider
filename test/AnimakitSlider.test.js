/* eslint-env jest */

import { render }    from 'enzyme';

import React          from 'react';
import AnimakitSlider from '../lib/AnimakitSlider.js';

describe('<AnimakitSlider />', () => {
  it('should render', () => {
    const component = render(<AnimakitSlider />);

    expect(component).toMatchSnapshot();
  });
});
