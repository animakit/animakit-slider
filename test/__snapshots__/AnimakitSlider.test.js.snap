// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AnimakitSlider /> should render 1`] = `
<div
  style={Object {}}
>
  <div
    style={Object {}}
  />
</div>
`;
